cluster-autoscaler/cloudprovider/azure/azure_manager.go

```diff
@@ -26,9 +26,11 @@ import (
 
 	"github.com/Azure/go-autorest/autorest/azure"
 	"gopkg.in/gcfg.v1"
+	"k8s.io/klog"
+	azure2 "k8s.io/legacy-cloud-providers/azure"
```

Comment:
looks good to me in general. Could we rename azure2 to providerazure?

Reply:
Also, please group k8s.io packages togather

Reply:
Fixed. I also added a note to explain the new behavior. Please take a look.